docs/source/workflow.rst

```diff
@@ -0,0 +1,94 @@
+   to merge the branch you have made the changes on into the `master` branch. In the creation of the pull request you
+   will need to assign specific people to review the changes you have made before it can be merged into the master branch.
+5. Once the updates have been approved by the review/reviewers they will merge the development into the master branch.
+6. When the private master branch is updated you will need to talk to the ERSEM repo manager to merge those changes back
+   to the public repo.
+
+Public developments
+-------------------
+
+The following come under public developments:
+
+1. Documentation - new descriptions and updates to current documentation, these include new ERSEM tutorials or 
+   fixes to current docs
+2. Bug fixes - these require immediate attention when identified within the code base
+3. Public project developments - if possible, small developments and additions to the code shall be added publicly,
+   these could include different options or parameterisations of the model.
+
+Workflow for public developments
+++++++++++++++++++++++++++++++++
+
+For public developments, the workflow is pretty simple:
+
+1. Create an issue or issues in the `public ERSEM <https://github.com/pmlmodelling/ersem>`_ 
+   repo based on the development required.
+2. Create a `branch <https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-and-deleting-branches-within-your-repository>`_
+   from the `public ERSEM <https://github.com/pmlmodelling/ersem>`_ repo. The naming
+   convention of the branch is as follows `XX-[description]` where `XX` is the issue ID (this can be found
+   after creation of the issue) and `[description]` is a one or two work description to identify the branch
+   with the corresponding issue.
+3. Commit all changes outlined in the issue to this branch and push branch to the ERSEM repo.
+4. Create `pull request <https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request>`_
+   to merge the branch you have made the changes on into the `master` branch. In the creation of the pull request you
+   will need to assign specific people to review the changes you have made before it can be merged into the master branch.
+5. Once the updates have been approved by the review/reviewers they will merge the development into the master branch.
+6. When the public master branch is updated you will need to talk to the ERSEM repo manager to merge those changes back
+   to the private repo.
+
+Notes for reviewers
+-------------------
+
+When `reviewing pull requests <https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/about-pull-request-reviews>`_ there are a few things to check:
+
+1. Ensure that the new changes do not adversely effect ERSEM. **Note** some modifications will change the results
```

Comment:
we might want to "define" adversely effect and the threshold level for changes to go into ERSEM.
With a model so interconnected, it is likely that changing one aspect will have consequences somewhere else: when the reviewer should highlight these? any non-computational difference should be highlighted (unless the author of the change already discussed those in the pull request) or only when these create big issues (e.g. unrealistic simulations)?
Furthermore, the adverse effect might only appear in certain condition: is there a reference set-up that reviewers need to run? or is the author of the change supposed to give it?

Reply:
Unfortunately it is hard to define a threshold level for changes, maybe re-wording to ensure that people will start a conversation about the changes with other members of the group would be best?